internal/api/oidc/auth_request_converter.go

```diff
@@ -19,6 +19,7 @@ import (
 
 const (
 	amrPassword     = "password"
```

Comment:
should we not comment that this is deprecated?

Reply:
Yes think so too. I also have check the docs

Reply:
It's correct in the docs 😏 https://docs.zitadel.com/docs/apis/openidoauth/claims#standard-claims